ui-tests/pom.xml

```diff
@@ -75,7 +75,7 @@
                     <forkCount>3</forkCount>
                     <environmentVariables>
                         <!-- Read by ATH -->
-                        <JENKINS_VERSION>${jenkins.version}</JENKINS_VERSION>
+                        <JENKINS_VERSION>2.190.3</JENKINS_VERSION>
```

Comment:
is this desired and done on purpose?

Reply:
Yes it is. The thing is, the upstream update center is rotating versions of UC for LTS, and ATH will blow up when it depends on such a version (dozens of test failures since last master build with no code change because of that). So it starts receiving way too new plugins from the _latest_ failing the plugin installation. So I am bumping the version we do ATH with to avoid that.